### ISSUE TITLE
Some arguments of training ResNet50 for 100 epochs and 800 epochs are different

### ISSUE THREAD

**Issue by wanghonghui1998** [facebookresearch/dino#145]
Hi, thanks for your great work and making the codes available !
I am trying to reproduce the results of training ResNet50 for 100 epochs. Following Readme.md, I use this command:

```
python -m torch.distributed.launch --nproc_per_node=8 main_dino.py --arch resnet50 --optimizer sgd --lr 0.03 --weight_decay 1e-4 --weight_decay_end 1e-4 --global_crops_scale 0.14 1 --local_crops_scale 0.05 0.14 --data_path /path/to/imagenet/train --output_dir /path/to/saving_dir
```
I notice the training loss in the first few epochs is slightly lower than that in the logs you provided, as folloing:
```
{"train_loss": 11.023236959338854, "train_lr": 0.002998920820176666, "train_wd": 9.999999999999553e-05, "epoch": 0}
{"train_loss": 10.481061559072215, "train_lr": 0.008999160637915185, "train_wd": 9.999999999999553e-05, "epoch": 1}
{"train_loss": 9.875821199300859, "train_lr": 0.014999400455653705, "train_wd": 9.999999999999553e-05, "epoch": 2}
{"train_loss": 9.05594916623845, "train_lr": 0.020999640273392222, "train_wd": 9.999999999999553e-05, "epoch": 3}
{"train_loss": 8.457147812790913, "train_lr": 0.026999880091130742, "train_wd": 9.999999999999553e-05, "epoch": 4}
{"train_loss": 7.88318823581691, "train_lr": 0.03300011990886926, "train_wd": 9.999999999999553e-05, "epoch": 5}
{"train_loss": 7.2753067889945395, "train_lr": 0.03900035972660778, "train_wd": 9.999999999999553e-05, "epoch": 6}
{"train_loss": 6.484845979036473, "train_lr": 0.0450005995443463, "train_wd": 9.999999999999553e-05, "epoch": 7}
{"train_loss": 5.85175334027917, "train_lr": 0.0510008393620848, "train_wd": 9.999999999999553e-05, "epoch": 8}
{"train_loss": 5.482414653904432, "train_lr": 0.05700107917982333, "train_wd": 9.999999999999553e-05, "epoch": 9}
```
I don't know it's normal or not. And I notice that some arguments of training ResNet50 for 100 epochs use the default value, which is different from training for 800 epochs, such as teacher_temp, clip_grad and use_bn_in_head. I wonder if these default values are the best recipe for training for 100 epochs.
By the way, can you provide the linear evaluation logs or result of ResNets trained for 100 epochs?

**Comment by woctezuma** [comment]
> can you provide the linear evaluation logs or result of ResNets trained for 100 epochs?

Are these the logs which you are looking for?
- https://github.com/facebookresearch/dino/issues/84#issuecomment-880517768
